foreshadow/smart/all.py

```diff
@@ -239,7 +240,8 @@ def pick_transformer(self, X, y=None, **fit_params):
         if 0 < ratio <= self.threshold:
             return self._choose_simple(s.values)
         else:
-            return SerializablePipeline([("null", None)])
+            # return SerializablePipeline([("null", None)])
+            return Imputer()
```

Comment:
It turns out that if the column has no missing value, using the 3rd party FancyImputer will fail the execution since it has a hard coded requirement of must having at least one missing value. That's why we are checking the ratio. I'd suggest we keep it this way for now. I'll add a TODO.